modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm

```diff
@@ -124,7 +124,7 @@
 	icon = 'modular_skyrat/modules/novaya_ert/icons/turret_deployable.dmi'
 	icon_state = "living"
 	base_icon_state = "living"
-	stun_projectile = /obj/projectile/bullet/c27_54cesarzowa/rubber
+	stun_projectile = /obj/projectile/bullet/c27_54cesarzowa
```

Comment:
replacing a stun projectile with a non-stun variant, hm. Worth thinking on this a tad more